File: app/src/main/java/vergecurrency/vergewallet/helpers/utils/DataUtils.java
package vergecurrency.vergewallet.helpers.utils;



public final class DataUtils {


	public static byte[] reverse(byte[] array) {

		for(int i=0; i<array.length/2; i++){
			byte temp = array[i];
			array[i] = array[array.length -i -1];
			array[array.length -i -1] = temp;
		}
		return array;
	}
}

File: app/src/main/kotlin/vergecurrency/vergewallet/helpers/utils/DataUtils.kt
package vergecurrency.vergewallet.helpers.utils


object DataUtils {


    fun reverse(array: ByteArray): ByteArray {

        for (i in 0 until array.size / 2) {
            val temp = array[i]
            array[i] = array[array.size - i - 1]
            array[array.size - i - 1] = temp
        }
        return array
    }
}
